app/src/pages/inside/common/suiteTestToolbar/actionPanel/actionPanel.jsx

```diff
@@ -222,7 +232,7 @@ export class ActionPanel extends Component {
           {!listView &&
             !debugMode && (
               <div className={cx('action-button')}>
-                <GhostButton icon={HistoryIcon} disabled>
+                <GhostButton icon={HistoryIcon} onClick={this.onHistoryHandler}>
```

Comment:
onHistoryHandler - bad naming. Prefix "on" requires action ("onClick"). "Handler" - it's not action. 
"onClickHistory"